src/main/resources/messages_en.properties

```diff
@@ -658,6 +658,8 @@ list.id.required=ListId is required
 list.id.invalid=We have not found the list id {0} in the system
 list.invalid=ListId is not a list
 list.locked=List is locked
+list.add.columns.exceeded.maximun.allowed=The number of columns add exceeded the maximum allowed.
+list.add.variables.exceeded.maximun.allowed=The number of variables add exceeded the maximum allowed. For add more you should remove some columns.
```

Comment:
The number of added variables exceeds the allowed max of 50. Either remove entry variables or columns to proceed